cmux-tui/bindings/go/raw/client_test.go

```
@@ -402,11 +402,101 @@ func TestSocketResolutionFallsBackWhenUnixPathIsTooLong(t *testing.T) {
+	)
+	if path != want {
+		t.Fatalf("long session path = %q, want %q", path, want)
+	}
+	if !unixSocketPathFits(path) {
+		t.Fatalf("long session path exceeds sockaddr_un: %q", path)
+	}
+	if err := os.MkdirAll(filepath.Dir(path), 0o700); err != nil {
+		t.Fatal(err)
+	}
+	_ = os.Remove(path)
+	listener, err := net.Listen("unix", path)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)